### ISSUE TITLE
(Feature request) Option to make double clicking on tray icon open home folder

### ISSUE THREAD

**Issue by qcksilver** [canton7/SyncTrayzor#659]
It would be great if there could be an option to change what double clicking on the tray icon does. When I double click, I would like it to behave like most shared folder programs (dropbox, pcloud, sync etc...) do and open the syncthing home folder.

I've installed synctrazor on everyone in my household's computer, and some of them are not very tech savy, and if they double click
on synctrazor now, synctrazor opens and they see the syncthing web control which is a) overwhelming to them and not useful (to them) and b) not really a place I want them to be, they can do more harm than good there. The have used dropbox in the past, so opening their home folder is the behavior they expect.

Thanks!

**Comment by canton7** [comment]
What is the "Syncthing home folder"? You can configure multiple folders, and [at least half of users do so](https://data.syncthing.net/)

**Comment by qcksilver** [comment]
Sorry, I used the wrong term terminology, I meant the syncthing default folder, which is %UserProfile%\Sync which can be changed in the syncthing settings:
![image](https://user-images.githubusercontent.com/19847101/128611539-6a78d826-bebd-4fda-9f8e-18c8dd0b0026.png)

I've changed mine to %UserProfile%\Syncthing. So the behavior I would like is for synctrazor to open whichever folder the user has set as the default.

Alternately (and maybe this is easier to implement?) is in the SyncTrazor settings to have an option for manually setting a folder to open when the synctrazor tray icon is double clicked.

Could also have 3 double click options in the synctrazor settings:
a) open synctrazor (default behavior)
b) open syncthing default folder
c) open manually set folder

**Comment by canton7** [comment]
I can't tell which folder the "Default" folder is (if there even is one) -- it's just the folder which Syncthing creates by default, there's nothing special about it.

I could see a bit of UI to select the folder to open. You'd need to add a context menu item to open SyncTrayzor in that case.

Contributions welcome!

**Comment by qcksilver** [comment]
> I could see a bit of UI to select the folder to open. 

Yeah that would fine if the only 2 options were open synctrazor (default behavior) or manually set folder to open 

>You'd need to add a context menu item to open SyncTrayzor in that case.

True, good point! Definitely still want to be able to open SyncTrayzor if the default behavior for double clicking is changed.

>Contributions welcome!

Unfortunately I am not capable of doing this (I wish I was!). Hopefully someone else reads this who has the skill and time to work on this. This would really improve my workflow and my housemates workflow for using syncthing.

I know there is a limit to the amount of time you have to work on SyncTrayzor, I appreciate all the work you have done!
